src/pages/es/core-concepts/framework-components.md

```diff
@@ -0,0 +1,153 @@
+
+> ⚠️ *Solo los componentes de **Astro** (`.astro`) pueden contener componentes de múltiples frameworks.*
+
+```astro
+---
+// src/pages/MyAstroPage.astro
+// Ejemplo: Mezclando múltiples componentes del framework en la misma página.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+import MySvelteComponent from '../components/MySvelteComponent.svelte';
+import MyVueComponent from '../components/MyVueComponent.vue';
+---
+<div>
+  <MySvelteComponent />
+  <MyReactComponent />
+  <MyVueComponent />
+</div>
+```
+
+## Anidando componentes de framework
+
+Dentro de un componente Astro, también puedes anidar componentes de múltiples frameworks.
+
+```astro
+---
+// src/pages/MyAstroPage.astro
+import MyReactSidebar from '../components/MyReactSidebar.jsx';
+import MySvelteButton from '../components/MySvelteButton.svelte';
+---
+
+<MyReactSidebar>
+  <p>Aquí hay una barra lateral con texto y un botón.</p>
+  <MySvelteButton client:load />
+</MyReactSidebar>
+```
+
+> ⚠️ *Recuerde: los propios archivos de los componentes de framework (por ejemplo, `.jsx`, `.svelte`) no pueden combinar varios frameworks.*
+
+Esto le permite crear "aplicaciones" completas en su framework de JavaScript preferido y renderizarlas, a través de un componente principal, en una página de Astro. Este es un patrón conveniente para permitir que los componentes relacionados compartan estados o contextos.
+
+Cada framework tiene sus propios patrones para anidar: `children` props y [render props](https://reactjs.org/docs/render-props.html) para React y Solid; `<slot />` con o sin nombres para Svelte y Vue, por ejemplo.
+
+Nota: los componentes de Astro siempre se renderizan en HTML estático, incluso cuando incluyen componentes de framework que son hidratados. Esto significa que solo se pueden pasar props que no hacen ninguna renderización a HTML. Pasar los "render props" de React o los slots con nombre a los componentes del framework desde un componente de Astro no funcionará porque los componentes de Astro no pueden proporcionar la ejecución del cliente que requieren esos patrones.
+
+## ¿Puedo hidratar los componentes de Astro?
+
+Si intentas hidratar un componente Astro con un modificador `cliente:`, obtendrás un error.
+
+Los [componentes de Astro](/es/core-concepts/astro-components/) son componentes de solo maquetado a HTML sin ninguna ejecución del lado del cliente. Pero puede usar una etiqueta `<script>` en el maquetado del componente Astro para enviar JavaScript al navegador que se ejecuta en el ámbito global.
+
+📚 Obtenga más información sobre [`<scripts>` del lado del cliente en los componentes de Astro](/es/core-concepts/astro-components/#scripts-del-lado-del-cliente)
+
+[mdn-io]: https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API
+[mdn-ric]: https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback
+[mdn-mm]: https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia
```

Comment:
We could change these 3 URLs for the spanish version, changing `en-US` for `es`, what do you think?